### DISCUSSION TITLE
Please help to apply lora to oobabooga

### DISCUSSION THREAD

**Discussion by SS050505** [oobabooga/text-generation-webui#2751] [Q&A]
Hi, i have created a lora using oobaboga, but i can not apply it to the model at all, i am not sure if i am doing something wrong, here are the steps i have followed:

- Loading 7B model in 8Bit and start the training.

- Generated the Lora file.

- when i go back to the model tab and select the lora file and press apply lora, i get error message:

TypeError: PeftConfig.__init__() got an unexpected keyword argument 'dtype'

any help please, i am stuck in this since 2 days

appreciating any help or hint on the topic

![Screenshot 2023-06-18 203938](https://github.com/oobabooga/text-generation-webui/assets/136999487/48ffae19-dd1a-4cd5-a7d7-7d0bedabf743)

**Comment by harpomaxx** [comment]
related to #2703
